app/admin/teams.rb

```diff
@@ -1,12 +1,16 @@
 include ActiveAdminHelpers
 
 ActiveAdmin.register Team do
-  permit_params :name, user_ids: []
+  permit_params :name, :timesheet_enabled, user_ids: []
 
   form do |f|
     f.inputs 'Team Details' do
       f.input :name
       f.input :users, as: :check_boxes, collection: User.order(:email).pluck(:email, :id).map { |email, id| [email, id] }
+
+      # Add checkbox for timesheet_enabled
```

Comment:
If you're going to use a checkbox it might be useful to put in some sort of separator between this and the users.